pkg/mirrorregistries/generator.go

```diff
@@ -66,21 +66,23 @@ func extractLocationMirrorDataFromRegistries(registriesConfToml string) ([]Regis
 
 	registriesTree, ok := tomlTree.Get("registry").([]*toml.Tree)
 	if !ok {
-		return nil, fmt.Errorf("Failed to cast registry key to toml Tree")
+		// If /etc/containers/containers.conf exists but has no "registry" key we don't want to fail
+		// but we proceed as if the file did not exist at all.
+		return nil, nil
```

Comment:
Would this be better in `IsMirrorRegistriesConfigured`?

Reply:
Log if not?

Reply:
Given how those functions are called from around the code, I think it's the safest to return `false` in `IsMirrorRegistriesConfigured` when file exists but has no registries, but also to return nil without error in `extractLocationMirrorDataFromRegistries`. That's because there are calls around to both of those functions so I think it is safer to handle it twice

Reply:
This should never happen now, right?